runner/Dockerfile

```diff
@@ -64,6 +64,7 @@ RUN export ARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) \
   && tar xzf ./runner.tar.gz \
   && rm runner.tar.gz \
   && ./bin/installdependencies.sh \
+  && mv ./externals ./externalstmp \
```

Comment:
Makes sense!

This seems to also explain why we don't need to change `dindrunner.Dockerfile` - that one should already have `externals` in there and we don't need `externalstmp` there, as dockerd must be in the same container as the runner.